src/hotspot/share/classfile/placeholders.cpp

```diff
@@ -132,9 +132,6 @@ void PlaceholderEntry::add_seen_thread(JavaThread* thread, PlaceholderTable::cla
   SeenThread* threadEntry = new SeenThread(thread);
   SeenThread* seen = actionToQueue(action);
 
-  assert(action != PlaceholderTable::LOAD_INSTANCE || seen == NULL,
-         "Only one LOAD_INSTANCE allowed at a time");
-
```

Comment:
Unclear why this is removed? If disabling the new flag causes this fail then shouldn't the new flag form part of the condition?

Reply:
If you want, it seemed unnecessary clutter for the assert.

Reply:
I'd like to understand how the assert connects to the changes.

Reply:
Multiple threads can now create the LOAD_INSTANCE placeholder for the same class in the case of non-parallel capable class loader, so this assert is no longer valid for this case.